### ISSUE TITLE
Tests should have access to dev-dependencies in Cargo.toml

### ISSUE THREAD

**Issue by tmandry** [oli-obk/ui_test#282]
Currently they only have access to regular dependencies.

**Comment by oli-obk** [comment]
You can specify a custom Cargo.toml instead of reusing the one from your project. We rebuild it from scratch anyway.

I'm not sure how to get the dev dependencies, and it would clash with cross testing anyway

**Comment by tmandry** [comment]
Yeah, I did end up specifying a Cargo.toml, it's just awkward because you have to make a stub lib.rs.

I was hoping there would be an easy way of doing this. Feel free to close if not planned.

**Comment by oli-obk** [comment]
I want to be able to specify dependencies in the ui test .rs file, as the separate Cargo.toml was always a hack out of laziness
